src/main/scala/com/github/sstone/amqp/ConnectionOwner.scala

```diff
@@ -44,7 +43,7 @@ object ConnectionOwner {
    * @return an amqp uri
    */
   def toUri(cf: ConnectionFactory): String = {
-    "amqp://%s:%s@%s:%d/%s".format(cf.getUsername, cf.getPassword, cf.getHost, cf.getPort, cf.getVirtualHost)
+    "amqp://%s:**REDACTED**@%s:%d/%s".format(cf.getUsername, cf.getHost, cf.getPort, cf.getVirtualHost)
```

Comment:
Looking at this part again, it could be considered a breaking change since it no longer returns a uri that can be used to actually connect to RMQ (unless someone's password was literally `**REDACTED**`)

I can leave this change if you think it unlikely anyone would be using this method directly and if it doesn't violate your support/versioning policies. Or I can change the logging statements to not include the url and just say "connected to $host" or some such.

Reply:
> I can leave this change if you think it unlikely anyone would be using this method directly and if it doesn't violate your support/versioning policies. Or I can change the logging statements to not include the url and just say "connected to $host" or some such.

I think the log line is useful.
What do you think of simply keeping this method as it was and adding another with a different name, for logging purposes:
```
private def buildRedactedUri(cf: ConnectionFactory): String = {
  "amqp://%s:**REDACTED**@%s:%d/%s".format(cf.getUsername, cf.getHost, cf.getPort, cf.getVirtualHost)
}
```

Reply:
Works for me. Done.